web/errors/errors.go

```diff
@@ -59,20 +35,47 @@ func ErrorHandler(err error, c echo.Context) {
 		log.Errorf("%s %s %s", req.Method, req.URL.Path, err)
```

Comment:
I would have preferred to keep the old way to have full details of the error in the log message